providers/tokenbucket/examples_test.go

```diff
@@ -0,0 +1,33 @@
+package tokenbucket
+
+import (
+	middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2"
+	grpc_ratelimit "github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/ratelimit"
+	"github.com/juju/ratelimit"
+	"google.golang.org/grpc"
+)
+
+// Here hard-coded for simplicity sake, but in your application you should make this configurable.
```

Comment:
Let me know if this makes sense.
```suggestion
// Hard-coded for simplicity sake, but make this configurable in your application.
```